lua/go/snips.lua

```diff
@@ -141,7 +141,7 @@ local function set_query()
     return
   end
   query_is_set = true
-  vim.treesitter.set_query(
+  vim.treesitter.query.set(
```

Comment:
this will fail nvim-0.8.3.
Could you add `vim.fn.has('nvim-0.8.3') == 0` check?